app/src/main/java/fr/free/nrw/commons/CommonsApplication.java

```diff
@@ -140,11 +140,14 @@ public void onCreate() {
         cacheData  = new CacheController();
     }
 
-    protected RefWatcher setupLeakCanary() {
+    protected void setupLeakCanary() {
```

Comment:
If the signature of this is to be changed, TestCommonsApplication's counterpart would need to be changed as well.

Reply:
By the way, having a RefWatcher instance could be useful, although we don't make use of it right now. : https://github.com/square/leakcanary/wiki/FAQ#how-do-i-use-it